client/app/cloud/project/compute/loadbalancer/cloud-project-compute-loadbalancer.controller.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
```

Comment:
This file should be in ES6.  See other controller files in IPLB for exemple.

Reply:
You may also want to create some service to split controller logic / business logic.  See iplb as an exemple

Reply:
done